index.html

```diff
@@ -59,6 +59,31 @@ <h1 id="title">
         This is a work in progress.
       </p>
     </section>
+    <section>
+      <h2>
+        Introduction
+      </h2>
+      <p>
+        The HTML &lt;model&gt; element allows a website to embed interactive 3D
+        models as conveniently as any other visual media. Models are served as 
+        a standalone resource. The &lt;model&gt; element also has support for 
+        interaction and animation playback while presented within the page, 
+        and supports more immersive experiences, such as the stereoscopic 
```

Comment:
Perhaps suggest that this is stereoscopic/immersive is optional if the availble.

Reply:
Ah yes - I have now updated with the language we had actually gotten through internally!